docs/installation.rst

```diff
@@ -119,7 +100,7 @@ Ingestion tasks populate the database with version control push logs, queued/run
 
   .. code-block:: bash
 
-     (venv)vagrant@local:~/treeherder$ celery -A treeherder worker -B
+     (venv)vagrant@local:~/treeherder$ celery -A treeherder worker -B --concurrency 5
```

Comment:
I'm on the fence as to which is best. I think most people will just copy and paste (and then use bash history) - and many may not read the description we put, but will hopefully struggle to misunderstand "--concurrency". 

Let's land this as is for now to avoid spending too much time on this - we can always revisit if we don't switch to Foreman/supervisor in the short-medium term.